app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours_signed/CheckOpeningHoursSigned.kt

```diff
@@ -27,7 +28,7 @@ class CheckOpeningHoursSigned(
         nodes, ways with
           opening_hours:signed = no
           and (
-            $hasOldOpeningHoursCheckDateFilter
+            $hasOldOpeningHoursSignedCheckDateFilter
```

Comment:
How does this behave with current tagging, though? E.g.

```
shop=kiosk
opening_hours:signed=no
check_date:opening_hours=2022-02-02
```
?

Looks to me that for the tags above (as previously tagged with StreetComplete), this quest is currently not created. But it should, shouldn't it?

Reply:
(Would make sense to add a test case for this and see if it runs through or not)